.github/workflows/publish-beta-release.yml

```diff
@@ -0,0 +1,73 @@
+name: Publish Beta Release to Github Only
+
+on: 
+  workflow_dispatch:
+
+jobs:
+  save_artifacts:
+    name: 'Build VSIX Files'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '16.13.x'
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.ref }}
+      - run: npm install -g lerna
+      - run: npm install
+      - run: npm run compile
+      - run: npm run vscode:package
+      - name: Stage Artifacts
+        run: |
+          mkdir extensions
+          find packages -name "*.vsix" -type f -exec cp {} ./extensions \;
+          cp ./packages/salesforcedx-vscode/CHANGELOG.md ./extensions
+      - name: Upload Extensions
+        uses: actions/upload-artifact@v3
+        with:
+          name: VS Code Extensions
+          path: ./extensions/
+  
+  get_release_variables:
+    # store the RELEASE_VERSION from the parent package as an output that we can reference elsewhere
+    name: 'Get Release Environment Variables'
+    runs-on: ubuntu-latest
+    outputs:
+      RELEASE_VERSION: ${{ steps.getVersion.outputs.version }}
+    steps: 
+      - uses: actions/checkout@v3
+      - id: getVersion
+        run: echo "::set-output name=version::"$(node -pe "require('./packages/salesforcedx-vscode/package.json').version")""
```

Comment:
I had a rough time with some of these (quote marks especially).  https://github.com/salesforcecli/sfdx-cli/blob/3442c3c20b472d3ac40eeb1c624177a153468b11/.github/workflows/dockerBuildSlim.yml#L24 for 2 examples of something that I did get to work.

Reply:
Also, we're gonna do this one:

```
- uses: notiz-dev/github-action-json-property@2192e246737701f108a4571462b76c75e7376216
        id: packageVersion
        with:
          path: 'package.json'
          prop_path: 'version'
```

consumed like
```
   release_name: ${{ steps.packageVersion.outputs.prop }}
```

Reply:
Nice! That's probably a safer way to get at that value